docs/DeveloperGuide.adoc

```diff
@@ -122,21 +122,22 @@ image::ModelClassDiagram.png[]
 The `Model`,
```

Comment:
Figure 7 is a pretty good diagram, but some things to take note. Consider the 2 arrows pointing to UniqueCardList, the two 1s initially confused us as being 11 so it might be better if you separate them a little more. Also, we would love it if the diagram was smaller. Cheers!